packages/marshal/src/helpers/error.js

```diff
@@ -47,18 +47,19 @@ export const ErrorHelper = harden({
   canBeValid: (candidate, check) => {
     const reject = !!check && (details => check(false, details));
     // TODO: Need a better test than instanceof
-    if (!(candidate instanceof Error)) {
-      return reject && reject(X`Error expected: ${candidate}`);
-    }
+    return (
+      candidate instanceof Error ||
+      (reject && reject(X`Error expected: ${candidate}`))
+    );
+  },
+
+  assertValid: candidate => {
+    ErrorHelper.canBeValid(candidate, assertChecker);
     const proto = getPrototypeOf(candidate);
     const { name } = proto;
     const EC = getErrorConstructor(name);
-    if (!EC || EC.prototype !== proto) {
-      const note = X`Errors must inherit from an error class .prototype ${candidate}`;
-      // Only terminate if check throws
-      reject && reject(note);
-      assert.note(candidate, note);
-    }
+    (EC && EC.prototype === proto) ||
+      Fail`Errors must inherit from an error class .prototype ${candidate}`;
```

Comment:
Removing these `assert.note` invocations from `ErrorHelper.canBeValid` might be running afoul of [Chesterson's fence](https://en.wikipedia.org/wiki/G._K._Chesterton#Chesterton's_fence)—they will no longer be encountered in the special "accommodate errors that are not valid Passables" `if (ErrorHelper.canBeValid(passable)) { return $encodeError(passable, $encoder) }` tests at the top of encode{Passable,ToCapData,ToSmallCaps} functions. If they _should_ be encountered in those cases, I will move the checks back into `ErrorHelper.canBeValid` along with explanatory comments.